nixos/modules/services/finance/odoo.nix

```diff
@@ -112,11 +118,11 @@ in
     services.postgresql = {
       enable = true;
 
+      ensureDatabases = [ "odoo" ];
       ensureUsers = [{
         name = "odoo";
-        ensurePermissions = { "DATABASE odoo" = "ALL PRIVILEGES"; };
+        ensurePermissions."DATABASE odoo" = "ALL PRIVILEGES";
```

Comment:
I don't think these formatting changes are necessary. Or at least add them and any other non-critical module changes in a separate commit

Reply:
Thank you!